### PR TITLE
Add an easy way to create traces and run validation

### PR DESCRIPTION
**Pull request by achamayou**

### CODE REVIEW

File: tla/README.md [hunk 1]
```diff
@@ -5,3 +5,9 @@ This directory contains some formal specifications of CCF. For more information,
 You can also interact with these specifications using codespaces:
 
 [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=180112558&machine=xLargePremiumLinux&devcontainer_path=.devcontainer%2Ftlaplus%2Fdevcontainer.json&location=WestEurope)
+
+## Trace validation
+
+You can produce fresh traces quickly from the driver by running the `make_traces.sh` script from this directory.
+
+Calling the trace validation on, for example, the `startup` scenario can then be done with `JSON=../build/startup.ndjson ./tlc.sh consensus/Traceccraft.tla`.
```
Comment (heidihoward) [inline tla/README.md:13]:
```suggestion
Calling the trace validation on, for example, the `startup` scenario can then be done with `JSON=../build/startup.ndjson ./tlc.sh consensus/Traceccfraft.tla`.
```

File: tla/README.md [hunk 2]
```diff
@@ -5,3 +5,9 @@ This directory contains some formal specifications of CCF. For more information,
 You can also interact with these specifications using codespaces:
 
 [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=180112558&machine=xLargePremiumLinux&devcontainer_path=.devcontainer%2Ftlaplus%2Fdevcontainer.json&location=WestEurope)
+
+## Trace validation
+
+You can produce fresh traces quickly from the driver by running the `make_traces.sh` script from this directory.
+
+Calling the trace validation on, for example, the `startup` scenario can then be done with `JSON=../build/startup.ndjson ./tlc.sh consensus/Traceccfraft.tla`.
```
Comment (heidihoward) [inline tla/README.md:13]: I tested this out (fresh codespace from this PR) and `startup.ndjson` does not exist
image

**Review by heidihoward — APPROVED** [review]

File: tla/README.md [hunk 3]
```diff
@@ -5,3 +5,9 @@ This directory contains some formal specifications of CCF. For more information,
 You can also interact with these specifications using codespaces:
 
 [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=180112558&machine=xLargePremiumLinux&devcontainer_path=.devcontainer%2Ftlaplus%2Fdevcontainer.json&location=WestEurope)
+
+## Trace validation
```
Comment (heidihoward) [inline tla/README.md:9]: Might be worth putting this in the docs instead

Comment (achamayou) [reply tla/README.md:9]: Ok, I'm moving everything then.

**Pull request merged**